### ISSUE TITLE
Release v1.32.6

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#1500]
:robot: A dedicated issue to run tests and deploy release [v1.32.6](https://github.com/deckhouse/deckhouse/milestone/50).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | gcp | azure | openstack | yandex-cloud | vsphere | static`
  - `git_ref` is a tag or branch: `vX.Y.Z | test-X.Y.Z* | main | release-X.Y`
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `containerd | docker`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.19 | 1.20 | 1.21 | 1.22 | 1.23`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag or branch: `vX.Y.Z | test-vX.Y.Z* | main | release-X.Y`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.21'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.19
/e2e/use/k8s/1.21

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.19 and 1.21.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build modules FE' and 'build FE' jobs are finished.

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [d7b065](https://github.com/deckhouse/deckhouse/commit/d7b0654c3eca0103507f337a0ba14a9bee108b67) in branch **main**:
```
[log-shipper] Label filter (#1424)

Co-authored-by: Artem Kladov <artem.gen.kladov@gmail.com>
Co-authored-by: Konstantin Aksenov <konstantin.aksenov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [99e33c](https://github.com/deckhouse/deckhouse/commit/99e33c509fe621fe79cc3b8a9ed35d3d17ee14e0) in branch **main**:
```
[docs] Fix formatting (#1501)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [50ed99](https://github.com/deckhouse/deckhouse/commit/50ed99126970fdf3a1121d751ad3f4e62ea5e931) in branch **main**:
```
[docs] Add schema of Deckhouse installation in a private environment (#1509)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [2bf854](https://github.com/deckhouse/deckhouse/commit/2bf8546fbdfdfe250a7644830130c432962d24f6) in branch **main**:
```
[chore] Update canary deploy settings for the stable release channel (#1508)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [10c6b0](https://github.com/deckhouse/deckhouse/commit/10c6b0da2405d02a4ad352437fe0be9d80125985) in branch **main**:
```
[docs] Update security page (#1511)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [9d547a](https://github.com/deckhouse/deckhouse/commit/9d547ac3413d11e17a530313f3d83d1e52ccc258) in branch **main**:
```
[global] update base alpine images (#1374)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [eaae55](https://github.com/deckhouse/deckhouse/commit/eaae553824f09e4db2d248e005e293699dbd9185) in branch **main**:
```
[candi] added ssh allow list to cloud providers (#1301)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [307f0a](https://github.com/deckhouse/deckhouse/commit/307f0a76cfd67cd0289c6b93bae9b16062a1c8e8) in branch **main**:
```
[monitoring] UnsupportedContainerRuntimeVersion Docker 20.* Containerd 1.[5-6].* (#1506)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ec509c](https://github.com/deckhouse/deckhouse/commit/ec509cbbb53aea6ccd95ffde1d3172db020661d3) in branch **main**:
```
[docs] Minor fix (#1513)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [dbfa1f](https://github.com/deckhouse/deckhouse/commit/dbfa1f237a07930ac68c135cec8a2558d182f122) in branch **main**:
```
Update kubeconfig-generator icon for grafana (#1510)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [a4f1e4](https://github.com/deckhouse/deckhouse/commit/a4f1e47892f60972f1998da4aae23cda9464900b) in branch **main**:
```
[flant-integration] Remove unused 'team' field from schema (#1514)

* remove from schema
* remove config migration for "plan" field
* add example comments to migration
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [60ac35](https://github.com/deckhouse/deckhouse/commit/60ac35db20055c71d4fc020aad1e57d9b44c1641) in branch **main**:
```
[docs] Bump web-router and hide patch version number (#1518)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [57f025](https://github.com/deckhouse/deckhouse/commit/57f025936dac3cd9452e26ba92a7e83258cb2e9f) in branch **main**:
```
[helm] refactor deprecated api metrics (#1461)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
Co-authored-by: Maksim Nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [eedd7d](https://github.com/deckhouse/deckhouse/commit/eedd7d50791ef6580f986742a693651ad4a623fb) in branch **main**:
```
[dhctl] Use images for control-plane from Deckhouse registry instead public images (#1517)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [47eee6](https://github.com/deckhouse/deckhouse/commit/47eee6360d7ab458aad036fb4a4f8b52d7ebbdfc) in branch **main**:
```
[cloud-provider-azure] Update CSI to v1.17.0 (#1512)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c73c66](https://github.com/deckhouse/deckhouse/commit/c73c6667a458181aec3a3af1da42ef4fcd77b500) in branch **main**:
```
[ingress-nginx] protobuf-exporter add improper input validation (#1519)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ecd86c](https://github.com/deckhouse/deckhouse/commit/ecd86c64814c5b7a4bdd13cd2fe3d2ec3c49d1f9) in branch **main**:
```
[ceph-csi] Fix nodeSelector for csi-node pods in helm_lib (#1522)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [f2d546](https://github.com/deckhouse/deckhouse/commit/f2d5466269413924cf4f77f275047ece6cc937aa) in branch **main**:
```
[docs] Update structure scheme on the site (#1525)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [1dc629](https://github.com/deckhouse/deckhouse/commit/1dc62921f0b9d1fc90a2d78ff0b4425fe0cd7d6b) in branch **main**:
```
[upmeter] Fix missing init container with migrations for emptyDir (#1524)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [267408](https://github.com/deckhouse/deckhouse/commit/2674088aed3680af060881104b07eb6584199c2a) in branch **main**:
```
[node-manager] Fix unset variable error for static nodes (#1516)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [62fa72](https://github.com/deckhouse/deckhouse/commit/62fa7212f59275ad3414b173d552c20388d460f1) in branch **main**:
```
[openvpn] UDP (#1432)

Co-authored-by: Andrey Polovov <andrey.polovov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [e7fe5a](https://github.com/deckhouse/deckhouse/commit/e7fe5abe634a8141d9322569ae86b7125bbceff5) in branch **main**:
```
[extended-monitoring] List kube-apiserver cached objects (#1535)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [7d0bd2](https://github.com/deckhouse/deckhouse/commit/7d0bd21ea517af3b401d078f3e0b6e9fa8941fb8) in branch **main**:
```
[docs] Clarify highAvailability automatic discovery
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [465aa6](https://github.com/deckhouse/deckhouse/commit/465aa6234e266084ccd0f5a2d304e209d71ec1f6) in branch **main**:
```
[candi] Ubuntu 22.04 support (#1505)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [17c337](https://github.com/deckhouse/deckhouse/commit/17c3371cea14969b25ce77ef0240310693610f66) in branch **main**:
```
[cloud-provider-aws] add peering for without-nat and standard layouts (#514)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [53504c](https://github.com/deckhouse/deckhouse/commit/53504c3bdb1c25c9195f919cbd307e7b0721bfbd) in branch **main**:
```
[openvpn] remove openvpn.storageClass from deckhouse configmap (#1493)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [390aa1](https://github.com/deckhouse/deckhouse/commit/390aa1b93b493817b44fea3c39933431d31cdfde) in branch **main**:
```
[istio] Documentation refresh (#1281)

[istio] Refactor documentation

Co-authored-by: Kirill Kononovich <41591254+kirkonru@users.noreply.github.com>
Co-authored-by: Artem Kladov <artem.gen.kladov@gmail.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [0ccf0f](https://github.com/deckhouse/deckhouse/commit/0ccf0f46f50c6f899c534fe775bf92ea59222952) in branch **main**:
```
[docs] Fix the text of the articles in the section «Accessing cluster» (#1533)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [742261](https://github.com/deckhouse/deckhouse/commit/742261c8665c88d3f0fcb2348cf6fb91a6d64771) in branch **main**:
```
[chore] Add codeowner for documentation (#1544)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [5a438b](https://github.com/deckhouse/deckhouse/commit/5a438beed3fd632c77d3d80e941906c921106fd7) in branch **main**:
```
[docs] Add the proxy parameter support to the getting started in a private environment. (#1529)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3b9120](https://github.com/deckhouse/deckhouse/commit/3b9120ab2515e2ba9a68cc3b68e909a602165f34) in branch **main**:
```
[deckhouse] Fix global values OpenAPI specification (#1520)

[docs] Fix global values OpenAPI specification
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [cec685](https://github.com/deckhouse/deckhouse/commit/cec6857fa7b5851d67356465ce81581697840402) in branch **main**:
```
[log-shipper] Refactor configuration hook (#1475)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [630a93](https://github.com/deckhouse/deckhouse/commit/630a9354b61b1a7603b2d9a0c76f0e4e9c3b3242) in branch **main**:
```
[monitoring-kubernetes] Fix K8SKubeletDown alerts (#1546)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c48582](https://github.com/deckhouse/deckhouse/commit/c4858209ab0673aae3dee6000fcbb21203fd4156) in branch **main**:
```
[doc] Revert dont-use-public-control-plane-images parameter for closed envs getting started and add /install for dhctl image for closed envs getting started (#1547)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3601c4](https://github.com/deckhouse/deckhouse/commit/3601c4c54e1633e6665b929ce322d343999b5963) in branch **main**:
```
Re-generate changelog (#1502)

Co-authored-by: konstantin-axenov <konstantin-axenov@users.noreply.github.com>
```
